app/models/organization.rb

```diff
@@ -68,8 +68,17 @@ def upcoming_distributions
     distributions&.this_week&.count || 0
   end
 
+  # Computes full address string based on street, city, state, and zip, adding ', ' and ' ' separators
   def address
-    "#{street} #{city}, #{state} #{zipcode}"
+    s = ''
```

Comment:
This seems a bit complex. How about something like the following?

```ruby
[street,
city,
state,
zipcode].reject(&:blank?).join(", ")
```

Reply:
That's a great idea. I'd modify that a little so that there is no comma between the state and zip:

```
state_and_zip = [state, zipcode].reject(&:blank?).join(' ')
[street, city, state_and_zip].reject(&:blank?).join(', ')```